javascript/javascript1/week2/WeatherWear.js

```diff
@@ -0,0 +1,24 @@
+
+function getClothsToWear(temperature)
+{
+   const curTemperature = temperature;
+    switch(true)
```

Comment:
nice idea with the switch statement!

this can be simplified by just checking the lowest value rather than both values in the range.